### PR TITLE
LPS-180069 | Test Automation | Master

### PR DESCRIPTION
**Pull request by davisaints**
**JIRA Ticket: [LPS-180069](https://issues.liferay.com/browse/LPS-180069)**

### CODE REVIEW

**Comment by davisaints** [comment]
ci:test:sf

**Comment by andreefarias** [comment]
Is blocked temporarily, because maybe necessary to change some structure on testcase.

**Comment by andreefarias** [comment]
It has been temporarily **blocked** as we may need to change the structure of the testcase.

**Comment by timpak** [comment]
@andreefarias can you create a thread on Slack to explain why this is blocked? You can explain https://github.com/liferay-bpm-qa/liferay-portal/pull/649 in the thread as well

**Comment by davisaints** [comment]
ci:test:sf

**Review by andreefarias — APPROVED** [review]
Nice @davisaints!
LGTM! 🎉

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 1]
```diff
@@ -3271,6 +3271,54 @@ definition {
...
+
+			ObjectPortlet.selectCustomObject(label = "Account");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.assertObjectActions(
+				actionActive = "Yes",
+				actionLabel = "Custom Action",
+				status = "Success");
+		}
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:3319]: We can skip these steps for two reasons. One, we want to keep this test narrow in scope, to only check if the account entry was updated. Two, we can use a generic actions test to check if the Actions UI is updated. Does that make sense?

cc: @andreefarias

**Review by timpak — CHANGES_REQUESTED** [review]
Hey @davisaints, great job with this test! Can you take a look at my comment? Once you make the following changes, you can forward this.

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase (hunk 1 above)
Comment (davisaints) [reply modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:3319]: Yeah, thanks for the review!

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
ci:forward

**Pull request closed without merging**